### PR TITLE
Warnings fixes

### PR DESCRIPTION
**Pull request by alexmatson-da**
Fixes many more warnings, there are just two or three that I disabled linting for, need to think about more in depth. 

Also deletes commented code and removes the images directory that's no longer used.

### CODE REVIEW

File: ui/src/pages/network/Clearing.tsx [hunk 1]
```diff
@@ -57,16 +57,19 @@ export const ClearingServiceTable: React.FC<Props> = ({ services }) => {
     }
   };
 
+  // setClearingAccountName is never used, but clearingAccountName is. How does this work?
+  // eslint-disable-next-line
   const [clearingAccountName, setClearingAccountName] = useState('');
+
+  // setMarginAccountName is never used, but marginAccountName is. How does this work?
+  // eslint-disable-next-line
   const [marginAccountName, setMarginAccountName] = useState('');
```
Comment (alexmatson-da) [inline ui/src/pages/network/Clearing.tsx:66]: yeah i couldn't figure this one out... a real head scratcher!

Comment (alexmatson-da) [reply ui/src/pages/network/Clearing.tsx:66]: I think @alexgraham-da last worked on this if i recall right

File: ui/src/components/InputDialog/InputDialog.tsx [hunk 2]
```diff
@@ -78,9 +78,9 @@ export function InputDialog<T extends { [key: string]: any }>(props: InputDialog
             Confirm
           </Button>
           {!props.isInline && (
-            <a className="a2 cancel" onClick={() => props.onClose(null)}>
+            <Button className="a2 cancel" onClick={() => props.onClose(null)}>
```
Comment (Sarahb-da) [inline ui/src/components/InputDialog/InputDialog.tsx:81]: is this still `a` styling?

Comment (alexmatson-da) [reply ui/src/components/InputDialog/InputDialog.tsx]: now it does :D

File: ui/src/pages/network/Clearing.tsx (hunk 1 above)
Comment (alexgraham-da) [reply ui/src/pages/network/Clearing.tsx:66]: I see the issue...I changed accept offer for `Offer` to use `ClearingOfferModal`, but the `acceptOffer` function is still there for `MarketOffer` and I didn't remove the code for regular `Offer`.

If you change `acceptOffer` on line `111` to:

```typescript
const acceptOffer = async (c: CreateEvent<MarketOffer>) => {
  await ledger.exercise(MarketOffer.Accept, c.contractId, {});
};
```

you should be able to get rid of those variables.

**Review by Sarahb-da — APPROVED** [review]

**Review by alexgraham-da — APPROVED** [review]
looks good, thanks for this 👍

File: ui/src/pages/network/Clearing.tsx (hunk 1 above)
Comment (alexmatson-da) [reply ui/src/pages/network/Clearing.tsx]: That works!

**Pull request merged**